packages/calcite-components/src/components/inline-editable/inline-editable.tsx

```diff
@@ -69,13 +69,13 @@ export class InlineEditable extends LitElement implements InteractiveComponent,
   /** Specifies a callback to be executed prior to disabling editing via the controls. When provided, the component's loading state will be handled automatically. */
   @property() afterConfirm: () => Promise<void>;
 
-  /** When `true` and `editingEnabled` is `true`, displays save and cancel controls on the component. */
+  /** When present and `editingEnabled` is `true`, displays save and cancel controls on the component. */
```

Comment:
Would we want the same language for `editingEnabled`?

```suggestion
  /** When present and `editingEnabled` is present, displays save and cancel controls on the component. */
```